3.10/arangosync-migration-tool.md

```diff
@@ -0,0 +1,232 @@
+target deployment will be lost.
+{% endhint %}
+
+### During the migration...
+
+What happens during active migration:
+- The source data cluster remains usable. 
+- The target deployment in Oasis is switched to read-only mode.
+- Your root user password is not copied to the target deployment in Oasis.
+To get your new root password, select the target deployment from the Oasis
+Dashboard and go to the **Overview** tab. All other users are fully synchronized.
+
+{% hint 'warning' %}
+The migration tool increases the CPU and memory usage of the server you are
+running on. Depending on your ArangoDB usage pattern, it may take a lot of CPU
+to handle the replication. You can stop the migration process anytime
+if you see any problems.
+{% endhint %}
+
+During active migration, the agent HTTPS server is executed on the `$MG_HOST:8629` 
+endpoint. Make sure the host and port are available for the Oasis platform.
+To change the default port, use the `--agent.master-port` option.
+
+```bash
+export MG_HOST=<your IP or publicly-available hostname here>
+./arangosync-migration start \
+  --source.cacert=tls-ca.crt \
+  --source.keyfile=client-auth.keyfile \
+  --source.endpoint=$COORDINATOR_ENDPOINT \
+  --source.jwtSecret=clusterSecret \
+  --oasis.api-key=$OASIS_API_KEY \
+  --oasis.api-secret=$OASIS_API_SECRET \
+  --oasis.deployment-id=$OASIS_DEPLOYMENT_ID \
+  --agent.listen-host=$MG_HOST
+```
+
+### TLS server certificates
+
+The migration agent HTTPS server uses TLS certificate pairs to ensure a secure
+connection between your local cluster and the Oasis platform.
+If you do not provide them, the migration tool creates self-signed certificates.
+If you can provide TLS certificates, use the `arangodb` tool to convert them in
+a suitable format for the migration tool.
+See a detailed guide on how to [create a new certificate/keyfile pair](programs-starter-security.html).
+Make sure to specify your publicly available host name, `$MG_HOST` when creating
+the keyfile. 
+
+When starting the migration, specify the generated files on the command line:
+
+```bash
+--agent.cacert=tls.crt --agent.keyfile=tls.keyfile --agent.client-auth-cacert=client-auth-ca.crt --agent.client-auth-keyfile=client-auth.keyfile
+```
+
+### How long does it take?
+
+The total time required to complete the migration depends on how much data you
+have and how many write operations are executed during the process.
+
+You can also track the progress by checking the **Migration status** section of
+your target deployment in Oasis.
+
+![Oasis Cloud Migration Progress](images/oasis-migration-agent.png)
+
+### Getting the current status
+
+To print the current status of the migration, run the following command:
+
+```bash
+arangosync-migration status
+```
+
+You can also add the `--watch` option to start monitoring the status in real-time.
+
+### Stopping the migration process
+
+To stop the migration, run the following command:
```

Comment:
I think this sentence and short 'stop' can be removed from here, because there is full example a bit below. Also this short command will not work without parameters